data_juicer/ops/filter/image_text_matching_filter.py

```diff
@@ -0,0 +1,178 @@
+import numpy as np
+from jsonargparse.typing import ClosedUnitInterval
+
+from data_juicer.utils.availability_utils import AvailabilityChecking
+from data_juicer.utils.constant import Fields, StatsKeys
+from data_juicer.utils.mm_utils import SpecialTokens, load_image
+from data_juicer.utils.model_utils import get_model, prepare_model
+
+from ..base_op import OPERATORS, Filter
+from ..op_fusion import LOADED_IMAGES
+
+OP_NAME = 'image_text_matching_filter'
+
+with AvailabilityChecking(['torch'], OP_NAME):
```

Comment:
In the first argument of AvailabilityChecking, we also need to add `transformers` to the dependency list. Same in the image_text_similarity_filter OP.

Reply:
Done